help.go

```diff
@@ -286,12 +291,29 @@ func ShowCommandCompletions(ctx *Context, command string) {
 // The customFuncs map will be combined with a default template.FuncMap to
 // allow using arbitrary functions in template rendering.
 func printHelpCustom(out io.Writer, templ string, data interface{}, customFuncs map[string]interface{}) {
+
+	const maxLineLength = 10000
+
 	funcMap := template.FuncMap{
 		"join":    strings.Join,
 		"indent":  indent,
 		"nindent": nindent,
 		"trim":    strings.TrimSpace,
+		"wrap":    func(input string, offset int) string { return wrap(input, offset, maxLineLength) },
+		"offset":  offset,
+	}
+
+	if customFuncs["wrapAt"] != nil {
```

Comment:
I think it is better to define a type for this function signature.

Reply:
I'm not quite sure what you mean- are you suggesting that we export a `WrapAtFunc` type alias, and then only use `customFuncs["wrapAt"]` if it is created as a `WrapAtFunc` explicitly?

Reply:
Gentle ping on this :)

Reply:
Yes correct

Reply:
This requires some boilerplate for the caller, but I'm not sure how it makes things better. How does this help? I couldn't find any examples of this pattern in a quick web search, but I'm not sure what to call this so maybe I'm missing something obvious.

Reply:
fwiw I'm happy to defer resolution of this to a later PR, so I'm going to move forward with merging the current work 👍🏼